sys-block/arcconf/arcconf-2.06.23167.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI="7"
```

Comment:
Is this change really necessary?

Reply:
No, just did it, since all my other eBuilds have it this way ;-)

Reply:
>since all my other eBuilds have it this way

Some devs strip the quotation marks off when pushing your commit to gentoo tree (if you weren't aware), so isn't this like fighting a windmill? :)